be/src/runtime/load_channel_mgr.h

```diff
@@ -111,6 +120,9 @@ class LoadChannelMgr {
 
     // thread to clean timeout load channels
     bthread_t _load_channels_clean_thread;
+
+    // Thread pool used to handle rpc request asynchronously
+    std::unique_ptr<ThreadPool> _async_rpc_pool;
```

Comment:
most of the thread pools in BE are created and destroyed in the ExecEnv, thought it would be better move this rpc pool to there as well?

Reply:
The pool should be the internal of `LoadChannelMgr`, and  `LoadChannelMgr` is already a manager. I think it's better to put the pool here